components/TableOfContentListItem.tsx

```diff
@@ -0,0 +1,44 @@
+/* eslint-disable react/prop-types */
+import React, { Dispatch, SetStateAction } from 'react';
+import classNames from 'classnames';
+import { v4 } from 'uuid';
+import { TableContentModel } from '@models/TableOfConent.model';
+
+const TableOfContentListItem = ({
+    title,
+    active,
+    setActive
+}: {
+    title: TableContentModel;
+    active: string;
+    setActive?: Dispatch<SetStateAction<null | string>>;
+}): JSX.Element => {
+    return (
+        <li key={v4()}>
```

Comment:
This doesn't need a key because keys, check the doc on when use keys: https://reactjs.org/docs/lists-and-keys.html